netty/src/test/java/io/grpc/transport/netty/NettyClientHandlerTest.java

```diff
@@ -135,10 +145,28 @@ public void setUp() throws Exception {
   }
 
   @Test
-  public void createStreamShouldSucceed() throws Exception {
-    handler.write(ctx, new CreateStreamCommand(grpcHeaders, stream),
-        promise);
+  public void cancelBufferedStreamShouldChangeClientStreamStatus() throws Exception {
+    // Force the stream to be buffered.
+    setMaxConcurrentStreams(0);
```

Comment:
Can you rename `setMaxConcurrentStreams()` to something like `receiveMaxConcurrentStreams()` to distinguish it from just setting the value via `handler.connection().local().maxActiveStreams()`?